src/Middleware/Diagnostics/src/DeveloperExceptionPage/DeveloperExceptionPageMiddlewareImpl.cs

```diff
@@ -101,6 +101,19 @@ public async Task Invoke(HttpContext context)
         }
         catch (Exception ex)
         {
+
+            if (ex is OperationCanceledException operationCanceledException && context.RequestAborted.IsCancellationRequested)
+            {
+                _logger.LogDebug("Request aborted");
```

Comment:
Add a new loggermessage for this so it gets all the right metadata:
https://github.com/dotnet/aspnetcore/blob/04bb460447cc6b84ec88080a777b298ba70ce7d7/src/Middleware/Diagnostics/src/DiagnosticsLoggerExtensions.cs#L8-L15

Reply:
I have checked in the changes. Please let me know if you need any change @Tratcher